### ISSUE TITLE
Community Engagement and products

### ISSUE THREAD

**Issue by Maraket** [evolus/pencil#679]
I have used Pencil on and off for a few years now, I'm not artistic but I find it useful to mockup simple wireframes (nothing like the super polished mockups people can do, I'm talking lines and boxes, more doodles) but still find it incredibly useful.

I'm wondering several things:

- Is there a Pencil discord?
- Is there a pencil forum?
- The current https://pencil.evolus.vn/ though capable is somewhat dated and could use a refresh
- Accessing "stencils and templates" is hosted on Google code archive, are there plans to move it to something like github
- Has there been any investigation to migrating documentation to something like Atlassian, or another cloud solution that offers open source projects free use of some of their services, having something like a JIRA board would likely give devs who are interested in helping the ability to more clearly co-ordinate work as well as help to give clear indications of roadmaps.

**Comment by Jeeppler** [comment]
Pencil is very useful for wire framing.

**Comment by Maraket** [comment]
@Jeeppler I agree, that is why I am trying to gauge where the current community engagement is, and what is being done to try and leverage the community and make it more popular. 
I see similarities between Pencil and FIGMA and notice that some of the reasons that FIGMA is more popular then Pencil boils down to the community engagement and resources, which is sad given Pencil as I understand it is older then FIGMA

**Comment by Jeeppler** [comment]
@Maraket I would like to have a free and open source software tool. Pencil is that.

**Comment by Maraket** [comment]
@Jeeppler I agree with you, and I prefer free and open source tools as they tend to be more ethical and can have a richer experience due to their open source nature. The issue I am trying to highlight/address is what I see as a major weakness of Pencil vs competing solutions (close sourced or otherwise) and that being there is virtually no community

**Comment by Mannshoch** [comment]
Ping @mbrainiac

**Comment by pavinduLakshan** [comment]
Currently, there is no user manual for this software as well. That makes users abandon the software when they face a problem and can't find docs or manuals for the software to fix the issue themselves. It would be great if we can host a Github Pages site for this repository and include the documentation there. 

I would like to work on this only if I'm confident that it will not be stayed forever as a PR or on dev branch🙂

**Comment by Mannshoch** [comment]
May @mbrainiac is no longer interested in this project and Pencil is kind of a zombie.

**Comment by Jeeppler** [comment]
Maybe, the way forward is to fork it and continue the development. The software itself is useful and basing version 3+ on electron was a major improvement.

**Comment by alrama** [comment]
[https://nathanielw.github.io/pencil-stencils/](https://nathanielw.github.io/pencil-stencils/) publish some more stencils for Pencil. For example Android Lollipop and Material Icon stencils enable an use much more modern

**Comment by javirosa** [comment]
I feel similarly. I'd like to help, but I don't know where to start. I'm actually interested in addressing this ancient bug (old repos issue tracker), but I don't feel like working on if it that code doesn't have any hope of going anywhere: https://github.com/prikhi/pencil/issues/537

**Comment by dumblob** [comment]
@javirosa and everybody here - how about creating a new GitHub organization with relevant repos (most recent copy of this repo and https://github.com/mbrainiac/stencils-repository at the least) and continuing there (with wiki page turned on - i.e. anybody can contribute)?

I have done this already several times with different open source projects and all of them survived thanks to this move. If @evolus would be OK with it, we could avoid the fork and just move the repos under the new org (preserving all issues, comments, etc. without any hassle).

First thing is to ensure significantly increasing the [bus factor](https://en.wikipedia.org/wiki/Bus_factor ) by giving admin privileges to everybody who did at least a small thing in the past (PR review, PR itself, end-to-end complex bug report and testing, community moderation, wiki editing, ...).

Second step would be to contact everybody who has forked these repos and ask for commiting their changes to the newly forked repo (incl. potential relicensing and maybe even copyright waiving). Also additional stuff like https://nathanielw.github.io/pencil-stencils/ .

Third step would be to revamp the web page (and serve it from under the new organization's wings - perhaps pencil.github.io ) - ideally @evolus would then archive these old repos.

I am hereby proposing a deadline for the organization creation ~1 month from now: 7. Dec 2022

**Comment by Mannshoch** [comment]
Nice idea.
Personally I switched to [Draw.io](https://github.com/jgraph/drawio-desktop). Only thing I do not like on it are the "Stencils". These from Pencil are interactive, but these in Draw.io are not.
